WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt

```diff
@@ -181,7 +187,22 @@ class HelpActivity : LocaleAwareActivity() {
         AnalyticsTracker.track(Stat.SUPPORT_HELP_CENTER_VIEWED)
     }
 
+    private fun showMigrationFaq() {
+        val intent = Intent(Intent.ACTION_VIEW, Uri.parse("https://jetpack.com/support/switch-to-the-jetpack-app/"))
+        startActivity(intent)
+        AnalyticsTracker.track(Stat.SUPPORT_MIGRATION_FAQ_TAPPED)
+    }
+
     private fun HelpActivityBinding.showContactUs() {
+        lifecycleScope.launchWhenStarted {
+                if (jetpackMigrationStore.migrationComplete() == MigrationCompleteFetchedPayload.Success) {
```

Comment:
I'm not sure why making just a couple of calls resulted in error 429. However, instead of fetching `migrationComplete()`, we can use the value in the storage. The [AppPrefs.isJetpackMigrationCompleted()](https://github.com/wordpress-mobile/WordPress-Android/blob/fbb7cea952d4320586a9f77a44eaeba7bf39d31b/WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefsWrapper.kt#L257) function seems to be good for our case here.

Reply:
I was just making that change to app prefs 😄